tests/regression/deploy/test_deploy_regression.py

```diff
@@ -67,6 +67,7 @@ def test_deploy_with_all_args(self, template_file):
             "aws_stack_name": aws_stack_name,
             "sam_stack_name": sam_stack_name,
             "capabilities": "CAPABILITY_IAM",
+            "s3_bucket": self.s3_bucket.name,
```

Comment:
Curious, why did you add these? Trying to understand why this change required these to be added to the tests.

Reply:
So when I was working on this, we had the resolve-s3 feature merged, which needed either s3_bucket or -resolve-s3 parameter for deploy. But since last week, I pushed another PR which removed this check in deploy as it was not needed in deploy (only needed in package), which got merged yesterday. Thus I can now remove this as we don't need it.